src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/controller/employers/EmployersGetShould.kt

```diff
@@ -157,39 +154,39 @@ class EmployersGetShould : EmployerTestCase() {
 
     assertGetEmployersIsOkAndSortedByDate(
       parameters = "sortBy=createdAt",
-      expectedDatesSorted = listOf("2024-07-01T01:00:00Z", "2024-07-02T01:00:00Z"),
+      expectedSortingOrder = "asc",
     )
   }
 
   @Test
   fun `retrieve a default paginated Employers list sorted by creation date, in ascending order`() {
+    val sortingOrder = "asc"
     givenEmployersMustHaveDifferentCreationTimes()
 
     assertAddEmployerIsCreated(body = tescoBody)
     assertAddEmployerIsCreated(body = sainsburysBody)
 
     assertGetEmployersIsOkAndSortedByDate(
-      parameters = "sortBy=createdAt&sortOrder=asc",
-      expectedDatesSorted = listOf("2024-07-01T01:00:00Z", "2024-07-02T01:00:00Z"),
+      parameters = "sortBy=createdAt&sortOrder=$sortingOrder",
+      expectedSortingOrder = sortingOrder,
     )
   }
 
   @Test
   fun `retrieve a default paginated Employers list sorted by creation date, in descending order`() {
+    val sortingOrder = "desc"
     givenEmployersMustHaveDifferentCreationTimes()
 
     assertAddEmployerIsCreated(body = tescoBody)
     assertAddEmployerIsCreated(body = sainsburysBody)
 
     assertGetEmployersIsOkAndSortedByDate(
-      parameters = "sortBy=createdAt&sortOrder=desc",
-      expectedDatesSorted = listOf("2024-07-02T01:00:00Z", "2024-07-01T01:00:00Z"),
+      parameters = "sortBy=createdAt&sortOrder=$sortingOrder",
+      expectedSortingOrder = sortingOrder,
     )
   }
 
   private fun givenEmployersMustHaveDifferentCreationTimes() {
```

Comment:
Now, this function contains just a line, and it's acting like a kind of 'alias' that adapts two levels of abstraction: the one that works with Employers and the other, below, that works with traversal infrastructure, like controlling the time provider.

We can now adapt the function's name to express more about the behaviour. I like the name you put on your function, and maybe we can take part in it like:

`private fun givenEmployersHaveIncreasingIncrementByDayCreationTimes`

 The name is a bit long, but I'm sure we'll find a better name in the following refactors. Naming is always difficult :-) 

What do you think?

Reply:
Nice! Let me rename this